### DISCUSSION TITLE
Please stop changing detekt plugin config structure

### DISCUSSION THREAD

**Discussion by 0neel** [detekt/detekt#6196] [Ideas]
We check detekt plugin config into our repository to allow new developers setting up the plugin easier.
It's really annoying to update that file everty time a plugin update it released.
It also breaks the plugin configuration since the previous config is not migrated but simply ignored.

Currently we already have 3 variants of the identical config in that file:
<img width="674" alt="Screenshot 2023-06-08 at 18 12 08" src="https://github.com/detekt/detekt/assets/3177746/c3279b11-dd4d-43fc-91a4-54b045b4f5c2">

**Comment by BraisGabin** [comment]
That looks like maven. This project doesn't provide any maven plugin. You should be using a third party plugin.You should report that to that plugin.

**Reply by 0neel** [reply]
It's actually not Maven but an Android Studio's config generated by this plugin https://plugins.jetbrains.com/plugin/10761-detekt in the `.idea` folder of a project.

**Reply by 3flex** [reply]
There's a separate repo for the IntelliJ plugin: https://github.com/detekt/detekt-intellij-plugin

It would be worth raising over there.
